cluster/addons/dns/kube2sky/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
```

Comment:
You can't actually statically link some dynamic libs dependencies :) we stopped being able to statically link cAdvisor for example.